import-service/serverless.yml

```diff
@@ -0,0 +1,74 @@
+service: import-service
```

Comment:
I recommend you use the TypeScript version of the config file - it is more clear and convenient.

Reply:
I would prefer the YAML file since it looks more clear and less redundant, specifically for a such little project